lib/components/src/blocks/DocsPage.stories.tsx

```diff
@@ -1,74 +1,64 @@
 import React from 'react';
-import { Title, Subtitle, DocsWrapper, DocsContent } from './DocsPage';
-import * as storyStories from './Story.stories';
-import * as previewStories from './Preview.stories';
-import * as propsTableStories from './PropsTable/PropsTable.stories';
-import * as sourceStories from './Source.stories';
-import * as descriptionStories from './Description.stories';
+import { Title, Subtitle, DocsPageWrapper } from './DocsPage';
+import * as Story from './Story.stories';
+import * as Preview from './Preview.stories';
+import * as ArgsTable from './ArgsTable/ArgsTable.stories';
+import * as Source from './Source.stories';
+import * as Description from './Description.stories';
 
 export default {
   title: 'Docs/DocsPage',
-  component: DocsWrapper,
+  component: DocsPageWrapper,
   parameters: {
     layout: 'fullscreen',
   },
-  decorators: [
-    (storyFn) => (
-      <DocsWrapper>
-        <DocsContent>{storyFn()}</DocsContent>
-      </DocsWrapper>
-    ),
-  ],
 };
-
-export const withSubtitle = () => (
-  <>
+export const WithSubtitle = () => (
+  <DocsPageWrapper>
     <Title>DocsPage</Title>
     <Subtitle>
       What the DocsPage looks like. Meant to be QAed in Canvas tab not in Docs tab.
     </Subtitle>
-    {descriptionStories.text()}
-    {previewStories.single()}
-    {propsTableStories.normal()}
-    {sourceStories.jsx()}
-  </>
+    <Description.Text {...Description.Text.args} />
```

Comment:
Maybe you should do the default parameter binding trick for these stories?

Reply:
The default parameter binding trick doesn't seem to work. `<Description.Markdown />` seems to fill in props with `{}`

Reply:
Yeah I see what you mean, I guess react passes `{}` by default not `undefined` which breaks that pattern.

Reply:
😞

Reply:
So for react it's really `Story.defaultProps = Story.args` method the only way?